connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Herder.java

```diff
@@ -169,6 +170,21 @@
      */
     void resumeConnector(String connector);
 
+    /**
+     * Given connector name, returns the ConfigDef appropriately
+     * @param connName name of the connector
+     * @return ConfigDef of the connector
+     */
+    ConfigDef getConfigDef(String connName);
```

Comment:
again, not a dealbreaker, but generally we try to avoid `get` prefixes on getters in Kafka code. there are a few exceptions, especially due to code originally being developed outside kafka, but best to avoid where possible.

Reply:
Fair. Let's remove the get :)